### PR TITLE
homework3

### PR DESCRIPTION
**Pull request by najahizquierdo**
this file isnt pushing from my terminal :(

### CODE REVIEW

**Comment by najahizquierdo** [comment]
@dafrenchyman  I'm not sure why my files werent pushing from the terminal. It kept saying "up to date" when it was not. I uploaded them manually, hope thats cool!

File: homework2/baseballscripts.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    baseballDF.createOrReplaceTempView("t_last100Games")
+    baseballDF.persist(StorageLevel.MEMORY_AND_DISK)
+
+    baseballDF2 = (
+        spark.read.format("jdbc")
+        .option("url", url)
+        .option("dbtable", t_rollingAverage)
+        .option("user", user)
+        .option("password", password)
+        .option("driver", diver)
```
Comment (brian004900) [inline homework2/baseballscripts.py]: 47 and 59: driver is misspelled.(pre-commit hook)

File: homework2/baseballscripts.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+
+def main():
+    appName = "Baseball Scripts in PySpark"
+    spark = SparkSession.builder.master("local[*]").getOrCreate()
+    url = "jdbc:mysql://localhost:3306/baseball?permitMysqlScheme"
+    driver = "org.mariadb.jdbc.Driver"
+    user = " "
+    password = ""
+
```
Comment (brian004900) [inline homework2/baseballscripts.py]: Successfully connected MariaDB and spark.

File: homework2/baseballscripts.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        spark.read.format("jdbc")
+        .option("url", url)
+        .option("dbtable", t_rollingAverage)
+        .option("user", user)
+        .option("password", password)
+        .option("driver", diver)
+        .load()
+    )
+    baseballDF2.createOrReplaceTempView("t_last100Games")
+    baseballDF2.persist(StorageLevel.MEMORY_AND_DISK)
```
Comment (brian004900) [inline homework2/baseballscripts.py]: Tranformer has not created.

File: homework2/baseballscripts.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    url = "jdbc:mysql://localhost:3306/baseball?permitMysqlScheme"
+    driver = "org.mariadb.jdbc.Driver"
+    user = " "
+    password = ""
+
+    t_last100Games = spark.sql(
+        """
+      select game_id
+      , local_date
+      from game
```
Comment (brian004900) [inline homework2/baseballscripts.py]: I was not able to show game table and run all the spark.sql statement.

File: homework2/baseballscripts.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    """
+    )
+
+    t_rollingAverage.createOrReplaceTempView("t_rollingAverage")
+    t_rollingAverage.persist(StorageLevel.MEMORY_AND_DISK)
+
+    baseballDF = (
+        spark.read.format("jdbc")
+        .option("url", url)
+        .option("dbtable", t_last100Games)
```
Comment (brian004900) [inline homework2/baseballscripts.py]: Getting table through database after it was created.

File: homework2/baseballscripts.py [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    t_last100Games = spark.sql(
+        """
+      select game_id
+      , local_date
+      from game
+      order by local_date DESC LIMIT 100
+      """
+    )
+    t_last100Games.createOrReplaceTempView("t_last100Games")
+    t_last100Games.persist(StorageLevel.MEMORY_AND_DISK)
```
Comment (brian004900) [inline homework2/baseballscripts.py]: Good to store tables in memory and disk.

**Comment by dafrenchyman** [comment]
The above `force-push` is what made you lose all your data.

**Pull request closed without merging**